src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(value = EnumType.STRING)
+    @Column(name = "role")
+    private RoleName roleName;
+
+    public RoleName getRoleName() {
+        return roleName;
+    }
+
+    public void setRoleName(RoleName roleName) {
+        this.roleName = roleName;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public enum RoleName {
+        ROLE_USER,
+        ROLE_ADMIN
```

Comment:
I think prefix ROLE is redundand
```suggestion
        USER,
        ADMIN
```